README.md

```diff
@@ -304,6 +304,26 @@ Waits for element to show up on the page.
 
 ```
 
+
+### .waitForElementToContain(selector, callback)
+
+Waits for the element to have some content.
+
+```javascript
+'Has the google ad loaded': function(done)
+{
+   // need to wait for all the element to be present
+  this.waitForElement('[data-role="page_first_ad"]', function()
+  {
+    this.waitForElementToContain('[data-role="page_first_ad"]', function() {
+      assert.contains(this.$('[data-role="page_first_ad"]').find('div:first').attr('id'), 'google_ads_iframe_');
+      done();
+    });
+  });
+}
```

Comment:
Since it's accepting only selector (and not resolved jQuery object) we don't need to wait for the element first, your example could be reduced to:

```javascript
'Has the google ad loaded': function(done)
{
  this.waitForElementToContain('[data-role=page_first_ad]', function() {
    assert.contains(this.$('[data-role=page_first_ad]').find('div:first').attr('id'), 'google_ads_iframe_');
    done();
  });
}
```